recipes/bootstrap/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "bootstrap" %}
+{% set version = "3.3.7" %}
+{% set sha256 = "859db5ef33728c6bfbea0ae8891c5e928176b9a69478761fe8363e5e8d354945" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/twbs/bootstrap/archive/v3.3.7.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: True
```

Comment:
Should be `noarch: generic` I think; see [docs](https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#architecture-independent-packages).